src/components/todo-list/todo-list.js

```diff
@@ -6,37 +6,44 @@ import '../todo-list-item/todo-list-item';
 import css from './todo-list.css';
 
 class TodoListComponent extends LitElement {
-  
   constructor() {
     super();
 
     this.todos = [];
-    
-    document.addEventListener('deleteTodo', (event) => this.deleteTodo(event.detail));
-    document.addEventListener('completeTodo', (event) => this.completeTodo(event.detail));
+
+    document.addEventListener('deleteTodo', event =>
+      this.deleteTodo(event.detail)
+    );
+    document.addEventListener('completeTodo', event =>
+      this.completeTodo(event.detail)
+    );
   }
 
-  // get user input through attributes? 
+  // get user input through attributes?
   // https://github.com/ProjectEvergreen/todo-app/issues/7
-  static get properties() { 
-    return { 
+  static get properties() {
+    return {
       todos: Array
     };
   }
 
-  addTodo() {
+  addTodo(e) {
+    e.preventDefault();
```

Comment:
this looks like it includes changes from #22 ?  We should try and refrain from mixing changesets between branches as much as possible, e.g.  1 issue : 1 PR.

Reply:
You're right, I didn't even notice that.